custom-plugin-rules/refs-plugin/README.md

```diff
@@ -0,0 +1,158 @@
+# Inspect $refs usage in an OpenAPI description
+
+Authors:
+
+- [`@tatomyr`](https://github.com/tatomyr) Andrew Tatomyr (Redocly)
+
+## What this does and why
+
+Dealing with $refs might be quite a challenge sometimes.
+This plugin introduces a few rules that could give insights into how the $refs are used.
+
+### No $refs siblings
```

Comment:
Maybe no _unexpected_ $refs siblings? Because we do still allow the valid ones like `description`.